sdk/schemaregistry/schema-registry/review/schema-registry.api.md

```diff
@@ -0,0 +1,96 @@
+## API Report File for "@azure/schema-registry"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+
+// @public (undocumented)
+export interface SchemaId {
+    // (undocumented)
+    id: string;
+}
+
+// @public (undocumented)
+export class SchemaRegistryClient extends SchemaRegistryClientContext {
+    constructor($host: string, options?: SchemaRegistryClientOptionalParams);
```

Comment:
`host` param having a `$` prefix is weird to me

Reply:
@nguerrera,  Is the $ coming from the swagger? I don't recall our code generator deciding to add this on its own.

Reply:
It was, but it's now named endpoint (no $) as of latest swagger.